### PR TITLE
4486 bigtables

### PR DESCRIPTION
**Pull request by sluhtoja**
created own branch.

### CODE REVIEW

**Comment by sluhtoja** [comment]
kick jenkins

**Comment by sluhtoja** [comment]
kick jenkins

**Comment by EfimovVladimir** [comment]
May be screenshot will help. It from build 1690. 
http://ec2-54-175-192-115.compute-1.amazonaws.com:8080/job/Beaker%20PR/ws/test/screenshots/bigIntegerTablesTutorial1471505450880.png
Is it expected result for table like that screenshot?
image

**Comment by sluhtoja** [comment]
fixed the problem with less than 30 columns.
image

File: plugin/ipythonPlugins/src/dist/ipython/ipython.js [hunk 1]
```diff
@@ -247,7 +248,9 @@ define(function(require, exports, module) {
               evaluation.payload = "<pre>" + result + "</pre>";
             }
             finalStuff = evaluation;
-            bkHelper.timeout(doFinish, 250);
+            var duration = new Date() - start;
+            console.log("duration" + duration);
```
Comment (scottdraves) [inline plugin/ipythonPlugins/src/dist/ipython/ipython.js]: remove debug logging

File: plugin/ipythonPlugins/src/dist/ipython/ipython.js [hunk 2]
```diff
@@ -367,7 +370,7 @@ define(function(require, exports, module) {
           }
           if (finalStuff === undefined) {            
             finalStuff = evaluation;
-            bkHelper.timeout(doFinish,150);
+            bkHelper.timeout(doFinish,250);
```
Comment (scottdraves) [inline plugin/ipythonPlugins/src/dist/ipython/ipython.js:372]: this change should be applied to python3.js and all the other similar plugins (each place with the same delay).

**Comment by sluhtoja** [comment]
kick jenkins

**Comment by sluhtoja** [comment]
kick jenkins

**Comment by sluhtoja** [comment]
kick jenkins

**Comment by scottdraves** [comment]
closing in favor of https://github.com/twosigma/beaker-notebook/pull/4684

**Pull request closed without merging**